src/main.js

```diff
@@ -92,12 +96,31 @@ export class Draggable {
             const { which } = e;
 
             if ((which && which > 1) || this._ignoreMouse) {
+                bind(this.document, "contextmenu", preventDefault);
+                
+                this.cancelDrag(e);
+
+                this._cancelHandler(e);
+
                 return;
             }
 
-            bind(this.document, "mousemove", this._mousemove);
-            bind(this.document, "mouseup", this._mouseup);
-            this._pressHandler(e);
+            if (!this._clickMoveClick) {
+                bind(this.document, "mouseup", this._mouseup);
+                bind(this.document, "mousemove", this._mousemove);
+                bind(this.document, "contextmenu", preventDefault);
+                this._pressHandler(e);
+            } else {
+                if (this._isDragging) {
+                    bind(this.document, "mouseup", this._mouseup);
+                } else {
+                    bind(this.document, "mousemove", this._mousemove);
+                    bind(this.document, "keydown", this._keydown);
+                    this._pressHandler(e);
+                }
```

Comment:
We can reverse the condition:

```
            if (this._clickMoveClick) {
                if (this._isDragging) {
                    bind(this.document, "mouseup", this._mouseup);
                } else {
                    bind(this.document, "mousemove", this._mousemove);
                    bind(this.document, "keydown", this._keydown);
                    this._pressHandler(e);
                }
            } else {
                bind(this.document, "mouseup", this._mouseup);
                bind(this.document, "mousemove", this._mousemove);
                bind(this.document, "contextmenu", preventDefault);
                this._pressHandler(e);
             }
```
By doing this we ask for a positive flag - whether dragging is enabled in a `click-move-click` manner instead of asking if it is not enabled. We can add other specific conditions in the future and they will appear at the top - so it is better for you to see all specific cases - and then the "default" behavior is left as a fallback at the end.